rasa/core/broker.py

```diff
@@ -56,21 +60,71 @@ def publish(self, event: Dict[Text, Any]) -> None:
         raise NotImplementedError("Event broker must implement the `publish` method.")
 
 
+def create_rabbitmq_ssl_options(
+    rabbitmq_host: Text = None
+) -> Optional[pika.SSLOptions]:
+    """Create RabbitMQ SSL options.
+
+    Requires the following environment variables to be set:
+
+    RABBITMQ_SSL_CLIENT_CERTIFICATE - path to the SSL client certificate (required)
+    RABBITMQ_SSL_CLIENT_KEY - path to the SSL client key (required)
+    RABBITMQ_SSL_CA_FILE - path to the SSL CA file for verification (optional)
+
+    Details on how to enable RabbitMQ TLS support can be found here:
+    https://www.rabbitmq.com/ssl.html#enabling-tls
+
+    Args:
+        rabbitmq_host: RabbitMQ hostname
+
+    Returns:
+        Pika SSL context of type `pika.SSLOptions` if all three paths are provided,
+        else `None`.
+
+    """
+
+    ca_file_path = os.environ.get("RABBITMQ_SSL_CA_FILE", None)
```

Comment:
optional: `None` is not needed since it's default (same for the others)
```suggestion
    ca_file_path = os.environ.get("RABBITMQ_SSL_CA_FILE")
```